mypy/config_parser.py

```diff
@@ -143,6 +149,7 @@ def check_follow_imports(choice: str) -> str:
     'disable_error_code': try_split,
     'enable_error_code': try_split,
     'package_root': try_split,
+    'exclude': str_or_seq_of_strs_as_list_of_strs,
```

Comment:
This function name is a bit verbose. I'm open to suggestions.

Reply:
Agreed and don't have a supremely better idea. Maybe `into_list` - implying that `list` is the result, but not implying that splitting is happening.